web/source/dom/domDefaultOutput.ts

```diff
@@ -0,0 +1,39 @@
+namespace com.keyman.text {
```

Comment:
Wouldn't the namespace be `com.keyman.dom`?

Reply:
Kinda debated a bit on that... literally everything in the class is being done on `com.keyman.text`, so I went with the version that let me drop the namespace prefixing requirement.

I can change it back by adding a lot of `text.` prefixes everywhere in the file, if that's preferred.

Reply:
I got no preference